src/components/Navigation.js

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+import { NavLink } from 'react-router-dom';
+
+export default function Navigation({ expensesActive }) {
+  return (
+    <Wrapper>
+      <Expenses to="/">Expenses</Expenses>
+      <Balances to="/balances">Balances</Balances>
+    </Wrapper>
+  );
+}
+
+const Wrapper = styled.header`
```

Comment:
I am not sure if header is the right element here, while you place the navigation on the BalancesPage at the bottom of the page which also includes a header element.

Reply:
Good point. Again, duplicated a component and missed something.